src/main/java/FlyChat/Tasks/Task.java

```diff
@@ -0,0 +1,48 @@
+package FlyChat.Tasks;
+
+/**
+ * Represents the Tasks of the user.
+ */
+public class Task {
+    private boolean done;
+    private String text;
+
```

Comment:
I noticed that the subclasses extending from Task all have an formatStringforSaving. 
Perhaps, for the Task Class, you can use a template string which acts as a Constant. That way, all the inheriting classes can get this formatString, allowing for code reuse :)